appserver/tests/embedded/maven-plugin/websockets/src/main/webapp/javascripts/application.js

```diff
@@ -140,6 +140,6 @@ var rules = {
         elem.onclick = app.post;
     }
 };
-Behaviour.addLoadEvent(app.initialize);
-Behaviour.register(rules);
+// Behaviour.addLoadEvent(app.initialize);
+// Behaviour.register(rules);
```

Comment:
Why is this being commented out?

Reply:
I missed cleaning it up. This is with regards to comment on Row 12 where you suggested to try to remove the non essential piece of code from these files. To that effect, appserver/tests/embedded/maven-plugin/websockets/src/main/webapp/javascripts/behaviour.js too has been removed in the updated PR. The test works fine even after these removals.

Reply:
If the unneeded dependency has been removed, these lines can just be removed.